src/learner/learner/learner.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from server.frontendstructs import Document, UserDocument
+
+import jsonpickle
+import os
+
+
+def learn(documents_folder, min_grade):
+    users = get_users()
+
+    for file_name in os.listdir(documents_folder):
+        file_path = os.path.join(documents_folder, file_name)
+        file_content = open(file_path).read()
+
+        topicmodeller_document = jsonpickle.decode(file_content)
+
+        document = Document(topicmodeller_document.url, topicmodeller_document.title, None)
+
```

Comment:
Maybe specify parameter name when argument is 'None' or we don't know what it is